exercises/concept/high-score/.docs/instructions.md

```diff
@@ -0,0 +1,127 @@
+# Instructions
+
+In this exercise, you're implementing a way to keep track of the high scores for the most popular game in your local arcade hall.
+
+<!-- TODO
+    Decide if "edge cases" should be listed in the instructions or not.
+
+    By explicitely mentioning them, students may consider them ahead of time when they don't really need to, because the edge cases match the behaviour of the naive implementation anyway.
+
+    However, if students deviate from the naive implementation, they may find it frustrating that tests fail even if their implementation conforms to the instructions.
+-->
```

Comment:
This refers to edge cases such as "What happens if the score of a player that isn't on the board is reset?".

I'd prefer not mentioning them in the instructions for the reasons in the comment but I don't have a strong opinion on it and am happy to change it if you think it's better to cover every case in the instructions.